dev/tidb-in-kubernetes/scale-in-kubernetes.md

```diff
@@ -34,15 +34,14 @@ To perform a horizontal scaling operation:
     watch kubectl -n <namespace> get pod -o wide
     ```
 
-    When the number of Pods for all components reaches the preset value and all components are in the `Running` state, the horizontal scaling is completed.
+    When the number of Pods for all components reaches the preset value and all components go to the `Running` state, the horizontal scaling is completed.
 
 > **Note:**
 >
-> - The PD and TiKV components do not trigger scaling up and down operations during the rolling update.
-> - When the PD and TiKV components scale downcall the corresponding interface to take offline the PD and TiKV nodes being deleted. This involves data migration operations, so it might take some time to finish the process.
-When the TiKV component scales in, it calls the PD interface to mark the corresponding TiKV instance as offline, and then migrates the data on it to other TiKV nodes. During the data migration, the TiKV Pod is still in the `Running` state, and the corresponding Pod is deleted only after the data migration is completed. The time consumed by scaling in depends on the amount of data on the TiKV instance to be scaled in. You can check whether TiKV is in the `Offline` state by running `kubectl get tidbcluster -n <namespace> <release-name> -o json | jq '.status.tikv.stores'`.
+> - The PD and TiKV components do not trigger scaling in and out operations during the rolling update.
+> - When the TiKV component scales in, it calls the PD interface to mark the corresponding TiKV instance as offline, and then migrates the data on it to other TiKV nodes. During the data migration, the TiKV Pod is still in the `Running` state, and the corresponding Pod is deleted only after the data migration is completed. The time consumed by scaling in depends on the amount of data on the TiKV instance to be scaled in. You can check whether TiKV is in the `Offline` state by running `kubectl get tidbcluster -n <namespace> <release-name> -o json | jq '.status.tikv.stores'`.
 > - The PVC of the deleted node is retained during the scaling down process, and because the PV's `Reclaim Policy` value is set to `Retain`, the data can be retrieved even if the PVC is deleted.
```

Comment:
See [the Chinese version](https://pingcap.com/docs-cn/v3.0/tidb-in-kubernetes/scale-in-kubernetes/).

```suggestion
> - When the PD and TiKV components scale in, the PVC of the deleted node is retained during the scaling in process. Because the PV's `Reclaim Policy` value is set to `Retain`, the data can still be retrieved even if the PVC is deleted.
```